docs/guides/advanced/query-api.md

```diff
@@ -197,22 +194,21 @@ else:
 
 ### Additional Configuration
 
-The Logfire API supports various query parameters and response formats to give you flexibility in how you retrieve your data:
+The Logfire API supports various response formats and query parameters to give you flexibility in how you retrieve your data:
```

Comment:
why were these swapped lol

Reply:
to match the order the bullets occur below